src/main/kotlin/io/github/inductiveautomation/kindling/log/LogPanel.kt

```diff
@@ -120,13 +418,37 @@ class LogPanel(
             EDT_SCOPE.launch {
                 table.model = createModel(filteredData)
             }
+            updateFilterPaneUI()
+        }
+    }
+
+    private fun updateFilterPaneUI() {
+        filterPane.run {
+            listOf(
+                loggerNamesSidebar,
+                loggerLevelsSidebar,
+                loggerTimesSidebar,
+                loggerMDCsSidebar,
+            ).forEachIndexed { index, sidebar ->
+                sidebar?.let {
+                    updateBackgroundAt(it.isFilterApplied, index)
+                }
+            }
+        }
+    }
+
+    private fun JTabbedPane.updateBackgroundAt(condition: Boolean, index: Int) {
+        if (condition) {
+            setBackgroundAt(index, background.darker())
```

Comment:
background.darker() barely shows up on MacOS dark theme - any reason to go with that over a more 'prominent' color from the LaF?

Reply:
I just wanted to add something simple that would work across all of the themes. Honestly I didn't test on MacOS though, I can find something a from the LaF though.